### PR TITLE
[WIN32SS][NTGDI][FREETYPE] Add ASSERT(pOS2); to FillTM

### PR DESCRIPTION
**Pull request by katahiromz**
## Purpose

JIRA issue: N/A

### CODE REVIEW

File: win32ss/gdi/ntgdi/freetype.c [hunk 1]
```diff
@@ -2272,6 +2272,10 @@ FillTM(TEXTMETRICW *TM, PFONTGDI FontGDI,
         return;
     }
 
+    ASSERT(pOS2);
+    if (!pOS2)
```
Comment (HBelusca) [inline win32ss/gdi/ntgdi/freetype.c]: Just to know, have we already encountered fonts that do not contain any kind of "OS/2" data structure? What is it for by the way?

Comment (katahiromz) [reply win32ss/gdi/ntgdi/freetype.c]: `MS Sans Serif` is raster font collection (`*.fon`) that doesn't contains the OS/2 table.

Comment (katahiromz) [reply win32ss/gdi/ntgdi/freetype.c]: I had used a static analyzer `cppcheck` to improve the code. This analyzer warned `NULL` check for this code.

Comment (HBelusca) [reply win32ss/gdi/ntgdi/freetype.c]: > `MS Sans Serif` is raster font collection (`*.fon`) that doesn't contain the OS/2 table.

OK but then, would the assert be triggered when using the MS Sans Serif?

**Pull request merged**